ArmVirtPkg/PlatformCI/PlatformBuildLib.py

```diff
@@ -223,7 +223,7 @@ def FlashRomImage(self):
         if (self.env.GetValue("TARGET_ARCH").upper() == "AARCH64"):
             cmd = "qemu-system-aarch64"
             args = "-M virt"
-            args += " -cpu cortex-a57"                                          # emulate cpu
+            args += " -cpu max"                                          # emulate cpu

```

Comment:
`max` is a moving target, so I'd rather have something specific here. IIRC `neoverse-n2` should work here too.

Reply:
Sure I'll give it a test with neoverse-n2 and update the PR assuming that works.